publishing/s3publisher.py

```diff
@@ -195,3 +205,24 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, cache_control,
 
             delta = datetime.now() - start_time
             LOGGER.info(f'... done in {delta.total_seconds():.2f}s')
+
+
+def update_admin_config(filename, base_url, owner, repository, branch,
+                        auth_base_url, auth_endpoint, bucket_type):
+    config = None
+    try:
+        with open(filename) as f:
+            config = yaml.safe_load(f)
+            if config:
+                if 'backend' in config:
+                    if bucket_type == 'dedicated':
+                        config['backend']['repo'] = f'{owner}/{repository}'
+                        config['backend']['base_url'] = auth_base_url
+                        config['backend']['auth_endpoint'] = auth_endpoint
+                        config['backend']['branch'] = branch
```

Comment:
This may warrant some thought/discussion. From what I understand this will be used as the base branch from which all changes made in Netlify CMS will be merged. I'm trying to think through all of the possibilities:

- If a user opens this up from production, using the `editorial workflow` it will create branches, then merge them to master which seems :100 to me.
- If a user creates a branch explicitly, then uses Netlify CMS from that branch, the changes should be merged into that branch, and then eventually to master when that is merged so that also seems :100.
- Anything I am missing?

Since we are using the `editorial workflow`, Netlify CMS will be creating branches in the repository, which will trigger Federalist builds for those branches, but I don't think we want this. Can we exclude those branches from triggering builds in Federalist???

Reply:
sounds like you're good to go on this but want to exclude builds by netlify? is there a reason? will it be creating a ton of extra noise? after creating examples of this then there could be a way.

Reply:
I take this back, it may create noise, but we probably do want to see the deploy preview for in progress changes made via Netlify CMS...